package.json

```diff
@@ -16,24 +16,19 @@
     "registration"
   ],
   "author": {
-    "name": "julianlam",
-    "email": "julian@nodebb.org"
+    "name": "river",
+    "email": "revir.qing@gmail.com"
   },
   "contributors": [
     {
       "name": "Julian Lam",
       "email": "julian@nodebb.org",
       "url": "https://github.com/julianlam/nodebb-plugin-sso-github"
-    },
-    {
-      "name": "Eric Miao",
-      "email": "qjmiao@gmail.com",
-      "url": "https://github.com/qjmiao/nodebb-plugin-sso-github"
     }
   ],
   "license": "BSD-2-Clause",
   "bugs": {
-    "url": "https://github.com/julianlam/nodebb-plugin-sso-github/issues"
+    "url": "https://github.com/revir/nodebb-plugin-sso-github2/issues"
   },
   "readme": "",
   "readmeFilename": "README.md",
```

Comment:
This file can be reverted too, though feel free to keep your name in `contributors` 😃